src/mixins/validations.js

```diff
@@ -1,11 +1,29 @@
-import Ajv from 'ajv';
+import Ajv from "ajv";
+const ajv = new Ajv();
 
-export const validateMixin = {
+export default {
   methods: {
+    async validateSchema(schema) {
+      const validate = await ajv.validateSchema(schema);
+      return {
+        success: validate,
+        error: ajv.errorsText(validate.errors)
```

Comment:
Nice! You found the method to validate the schemas!! Thank you!

https://ajv.js.org/api.html#ajv-validateschema-schema-object-boolean

```
Errors will be available at `ajv.errors`
```

I don't think you can get errors from the `validate` const.

Reply:
It does return the error. since validate become an instance ajv after assigning the response of validateSchema, the error is being returned. 
test it

Reply:
This works. I would change `validate.errors` to `ajv.errors`.
Also what's interesting is the `ajv.compile` method in  `validateJsonDataAgainstSchema` does call `validateSchema`.
https://ajv.js.org/api.html#ajv-compile-schema-object-data-any-boolean-promise-any

Reply:
I saw that, but it's mean validate the schema only when we validate everything which is like validating an email field on submit instead of when entering it

Reply:
Why would you use ajv.errors instead?

Reply:
Because documentation says "Errors will be available at `ajv.errors`."

https://ajv.js.org/api.html#ajv-validateschema-schema-object-boolean

and my IDE prefers it with ajv.errors :) but I tested and it works so I don't mind and approved the PR.